src/pages/groupComparison/Mutations.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import { action, computed, observable, makeObservable } from 'mobx';
+import autobind from 'autobind-decorator';
+import Loader from 'shared/components/loadingIndicator/LoadingIndicator';
+import { convertToMutationMapperProps } from 'shared/components/mutationMapper/MutationMapperConfig';
+import { getGenomeNexusHgvsgUrl } from 'shared/api/urls';
+import { getServerConfig } from 'config/config';
+import GroupComparisonMutationMapper from './GroupComparisonMutationMapper';
+import { Mutation } from 'cbioportal-ts-api-client';
+import MutationMapperToolStore from 'pages/staticPages/tools/mutationMapper/MutationMapperToolStore';
+import GroupComparisonStore from './GroupComparisonStore';
+import _ from 'lodash';
+import {
+    MUTATIONS_NOT_ENOUGH_GROUPS_MSG,
+    MUTATIONS_TOO_MANY_GROUPS_MSG,
+} from './GroupComparisonUtils';
+import AsyncSelect from 'react-select/async';
+import {
+    numberOfLeadingDecimalZeros,
+    formatPercentValue,
+} from 'cbioportal-utils';
+import { countUniqueMutations } from 'shared/lib/MutationUtils';
+
+interface IMutationProps {
+    store: GroupComparisonStore;
+    mutations?: Mutation[];
+    filters?: any;
+}
+
+@observer
+export default class Mutations extends React.Component<IMutationProps, {}> {
+    private mutationMapperToolStore: MutationMapperToolStore;
+    private groupComparisonMutationMapper:
+        | GroupComparisonMutationMapper
+        | undefined;
+
+    constructor(props: IMutationProps) {
+        super(props);
+        this.mutationMapperToolStore = new MutationMapperToolStore(
+            this.props.mutations ||
+                _(this.props.store.mutationsByGroup.result!)
+                    .values()
+                    .flatten()
+                    .value(),
+            this.props.filters || {
+                groupFilters: _(this.props.store.mutationsByGroup.result!)
+                    .keys()
+                    .value()
+                    .map(group => ({
+                        group: group,
+                        filter: {
+                            type: 'GroupComparisonFilter',
+                            values: [group],
+                        },
+                    })),
+                filterAppliersOverride: {
+                    ['GroupComparisonFilter']: this.props.store
+                        .applySampleIdFilter,
+                },
+                countUniqueMutations: this.countUniqueMutations,
+            }
+        );
+    }
+
+    @computed
+    get showPercent() {
+        return this.groupComparisonMutationMapper
```

Comment:
Parent component should not reach into child component.  Ideally child componet should accept a callback as a prop, which fires when say the percentage is changed and allows the parent to update the shared state.